src/app/saga/manuscript-editor.saga.ts

```diff
@@ -15,7 +15,7 @@ import { cancelManuscriptExport, getManuscriptExportStatus, startManuscriptExpor
 import { PDF_EXPORT_ERROR, PDF_EXPORT_SUCCESS } from 'app/store';
 import { LocalStorageApi } from 'app/api/local-storage.api';
 
-const POLL_INTERVAL = 1000;
+const POLL_INTERVAL = 1500;
```

Comment:
Should probably ask Exeter (ignore me if you have already) what would be an appropriate polling rate. I'd lean myself to something more around the 5 - 10 second mark so as not to generate too much traffic.